hack/local-up-cluster.sh

```diff
@@ -823,12 +825,17 @@ EOF
       --v=${LOG_LEVEL} 2>&1 &
     PROXY_PID=$!
 
+    cat <<EOF > /tmp/kube-scheduler.yaml
+apiVersion: componentconfig/v1alpha1
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: ${CERT_DIR}/scheduler.kubeconfig
+featureGates: ${FEATURE_GATES}
```

Comment:
Has this been tested locally? It doesn't seem to match the spec for KubeSchedulerConfiguration: https://godoc.org/k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1#KubeSchedulerConfiguration